docs/_data/nakadi-event-bus-api.yaml

```
@@ -3110,6 +3120,58 @@ definitions:
         example: 'data.security.team'
 
 
+  Annotations:
+    type: object
+    description: >
+      # [Work in progress] Annotations of the Nakadi resource.
+
+      A Nakadi event-type or a subscription is considered as a Nakadi resource.
+      A Nakadi resource can have a set of annotations associated with it.
+      They follow the same rules of annotations in Kubernetes resources.
+        For keys:
+      > Annotations are key/value pairs. Valid annotation keys have two segments:
+      an optional prefix and name, separated by a slash (`/`). The name segment is
+      required and must be 63 characters or less, beginning and ending with an
+      alphanumeric character (`[a-z0-9A-Z]`) with dashes (`-`), underscores (`_`),
+      dots (`.`), and alphanumerics between. The prefix is optional. If specified,
+      the prefix must be a DNS subdomain: a series of DNS labels separated by dots
+      (`.`), not longer than 253 characters in total, followed by a slash (`/`).
+
+        For values:
+      Kubernetes don't have any restrictions for the values of annotations.
+      But Nakadi limits this to 1000 bytes at the moment, this can be relaxed in the future.
+
+    example:
+      nakadi-sql/query-input: "true"
+    additionalProperties:
+      type: string
```

I thought it's only a boolean flag, like:
```suggestion
    additionalProperties: true
```
Or am I missing something?

---

We can also specify the type of values allowed for additional properties.

---

Ah, OK.  But do we really want to restrict it to `string` only?  I think in K8s structured annotation values are allowed (but not for labels).

---

I see them as JSON encoded as string.